src/bluesearch/entrypoint/compute_embeddings.py

```diff
@@ -126,6 +126,14 @@ def run_compute_embeddings(argv=None):
         type=int,
         help="Number of processes to use",
     )
+    parser.add_argument(
+        "-s",
+        "--start-method",
+        default="forkserver",
+        choices=["fork", "forkserver", "spawn"],
+        type=str,
+        help="Multiprocessing starting method to be used.",
```

Comment:
Should we maybe flesh out the help message a bit more? Maybe something similar to the description in the `MPEmbedder`? Just to give an idea why this parameter matters.

Reply:
Good point! I will provide more information!

Reply:
Done in 69774a4